examples/contracts/llm-erc20.py

```diff
@@ -0,0 +1,52 @@
+import json
+from genvm.base.equivalence_principle import EquivalencePrinciple
+
+
+class WizzardOfCoin:
+    prompt_part_1: str = """You keep track of transactions between users 
+    and their balance in coins. 
+    The current balance for all users in JSON format is:"""
+
+    prompt_part_2: str = "The transaction to compute is:"
+
+    prompt_part_3: str = """Given the current balance in JSON format and 
+    the transaction provided, please provide the updated balances as a 
+    JSON object.
+    For every transaction, validate that the user sending the Coins has 
+    enough balance. If any transaction is invalid it shouldn't be processed. 
+    Update the balances based on the valid transactions only.
+    It is mandatory that you respond with only the JSON balances object, nothing 
+    else whatsoever. Don't include any other words or characters, your 
+    output must be only the balances JSON object"""
+
+    def __init__(self, initial_owner: str, total_supply: int):
+        super().__init__()
+        self.balances = {}
+        self.balances[initial_owner] = total_supply
+
+    async def send(self, amount: int, from_address: str, to_address: str) -> None:
```

Comment:
We currently don't have the "from_address" implicit in the call and we can't use it as a global variable inside our IC code. This will be added when the backend is modified accordingly

Reply:
Understood about the arguments, maybe just rename to transfer then?

Reply:
Done!